packages/main/src/Toast.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="ui5-toast-root-parent"
+     dir="{{rtl}}">
+	<div class="ui5-toast-root"
+	     placement="{{placement}}"
+	     style="{{styles.root}}"
+		 @transitionend="{{_ontransitionend}}">
```

Comment:
You seem to have some spaces in the HBS file, please check and correct (use tab only).